packages/date-picker/src/vaadin-date-picker.js

```diff
@@ -162,6 +162,7 @@ class DatePicker extends DatePickerMixin(InputControlMixin(ThemableMixin(Element
         theme$="[[__getOverlayTheme(_theme, _overlayInitialized)]]"
         on-vaadin-overlay-open="_onOverlayOpened"
         on-vaadin-overlay-close="_onOverlayClosed"
+        focus-trap
```

Comment:
I wonder if enabling the `focus-trap` feature is really needed? As far as I see, the propagation of <kbd>Tab</kbd> that is fired inside the date-picker already seems to be stopped so that it shouldn't bubble to the document's `keydown` listener of `FocusTrapController`. Besides, removing `focus-trap` doesn't break any tests.

Reply:
Removed `focus-trap` from this PR and changed to use `event.stopPropagation()` as previously.